### PR TITLE
[openweathermap] Fix display of alert data

### PR DESCRIPTION
**Pull request by boehan**
Signed-off-by: Hans Böhm <h.boehm@gmx.at>

This fixes an issue where the last alert in the API response is always ignored. Also discussed in [the forum](https://community.openhab.org/t/openweathermap-new-one-call-api/101419/45).

### CODE REVIEW

File: bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java [hunk 1]
```diff
@@ -681,7 +681,7 @@ private void updateAlertsChannel(ChannelUID channelUID, int count) {
         List<Alert> alerts = localWeatherData != null ? localWeatherData.alerts : null;
         State state = UnDefType.UNDEF;
         if (alerts != null && alerts.size() > count) {
-            Alert alert = alerts.get(count - 1);
+            Alert alert = alerts.get(count);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java:684]: Please rename parameter `count` into `index` and update the javadoc description to mention that the expected value is from 0.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java (hunk 1 above)
Comment (boehan) [reply bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java:684]: updated the javadoc (also for the other methods)

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by boehan** [comment]
Just noticed that javadoc for `updateMinutelyForecastChannel` was missing --> added

**Comment by lolodomo** [comment]
Thank you

**Pull request merged**